### PR TITLE
Restapi plan

### PR DESCRIPTION
**Pull request by shemau**
### Description

Add a new feature to upgrade the container registry plan to standard.

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [x] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

##### Release notes content

Introduce a submodule that can upgrade the registry plan in a region (or global).

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- When merging, use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author. The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).
- Merge by using "Squash and merge".

### CODE REVIEW

File: modules/plan/README.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+<!-- BEGIN MODULE HOOK -->
```
Comment (ocofaigh) [inline modules/plan/README.md:1]: Can you remover the `<!-- BEGIN MODULE HOOK -->` - it does nothing

File: modules/plan/README.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  container_registry_endpoint = "us.icr.io"
+}
+```
+
+## Required IAM access policies
+
+You need the following permissions to run this module.
+
+
+TODO: validate
```
Comment (MatthewLemmond) [inline modules/plan/README.md:53]: remove TODO/validate policies needed

**Review by MatthewLemmond — CHANGES_REQUESTED** [review]
Small readme changes, but RestAPI provider setup/module looks good to me

File: modules/plan/README.md (hunk 2 above)
Comment (shemau) [reply modules/plan/README.md]: Resolved in latest commit

File: modules/plan/README.md (hunk 1 above)
Comment (shemau) [reply modules/plan/README.md]: Resolved in latest commit

**Review by MatthewLemmond — APPROVED** [review]
LGTM

**Review by MatthewLemmond — APPROVED** [review]

**Pull request merged**